### PR TITLE
Animals table, data and queries with results images

### PR DESCRIPTION
**Pull request by chepkok3**
- [ ] Created a database named vet_clinic.
- [ ] Created a table named animals with the following columns:

- id: integer
- name: string
- date_of_birth: date
- escape_attempts: integer
- neutered: boolean
- weight_kg: decimal

- [ ] Wrote queries for the following quests:

- All animals whose name ends in "mon".
- All animals born between 2016 and 2019.
- All animals that are neutered and have less than 3 escape attempts.
- All animals named either "Agumon" or "Pikachu".
- Name and escape attempts of animals that weigh more than 10.5kg
- All animals that are neutered.
- All animals not named Gabumon.
- All animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

- [ ] Screenshot of the results of the queries 
Find all animals whose name ends in "mon":

animals whose name ends in  mon

List the name of all animals born between 2016 and 2019.

name of all animals born between 2016 and 2019

List the name of all animals that are neutered and have less than 3 escape attempts.

name of all animals that are neutered and have less than 3 escape attempts

List the date of birth of all animals named either "Agumon" or "Pikachu".

date of birth of all animals named either  Agumon  or  Pikachu

List name and escape attempts of animals that weigh more than 10.5kg

name and escape attempts of animals that weigh more than 10 5kg

Find all animals that are neutered.

all animals that are neutered

Find all animals not named Gabumon.
animals not named Gabumon

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
animals with a weight between 10 4kg and 17 3kg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,243 +12,109 @@ After you're finished please remove all the comments and instructions!
...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
 I would like to thank...
```
Comment (addod19) [inline README.md]: - [ ] It would be better if we complete the acknowledgment section instead of leaving it uncompleted

File: README.md [hunk 2]
```diff
@@ -49,243 +12,109 @@ After you're finished please remove all the comments and instructions!
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (addod19) [inline README.md]: - [ ] It would be better if we fix our MIT License link, it is broken and returns `404` page

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @chepkok3, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Correct Git-flow followed  :+1: 
- [ ] Animals table is well structured to match the design :rocket: 
- [ ] Data insertion is done in the `data.sql` file :1st_place_medal: 


## Required Changes ♻️
_Check the comments under the review._
- [ ] It would be better if we add the screenshots of the results of our queries in the PR description
Screenshot from 2023-04-04 14-36-43


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @chepkok3👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**